ci/ReleaseTool/PrepCommand.cs

```diff
@@ -103,13 +88,11 @@ public PrepCommand(Options options)
         public int Run()
```

Comment:
Sadly can't generalise this whole function easily as there are small but important differences, such as the checking if a pre-existing branch exists (it is guaranteed to exist in the other step)